modules/data.land/R/helpers.R

```diff
@@ -0,0 +1,7 @@
+stop <- function(...) {
```

Comment:
My thinking behind these functions is they're a way to remove the dependency on `utils`. I always thought it was a bad idea to have packages depend on all of `utils` (which is huge, and has a lot of its own dependencies) just for the `logger` functions. So my approach is to use it only if it's installed (`requireNamespace`), and if not, to fall back on the standard R messaging functions. But that's just me -- happy to change this if folks think I should.